tests/wallet-automation/typhon/seed-phrase.ts

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import path from 'path';
+
+// Construct the path to the .txt file
+const txtPath = '../../wallet-automation/typhon/typhon-wallet-storage.txt';
```

Comment:
You should use a relative path, if you hardcode the path like this it will not work on someone else machine it should be instead

`const txtContent = fs.readFileSync(path.resolve(__dirname,'typhon-wallet-storage.txt'), 'utf8');`

This is the same for all the other paths in the code

Reply:
changed all file paths to the format you've suggested, kindly check =)